core/model/src/main/java/org/eclipse/rdf4j/TransactionSettingImpl.java

```diff
@@ -0,0 +1,10 @@
+package org.eclipse.rdf4j;
+
+public interface TransactionSettingImpl extends TransactionSetting {
```

Comment:
I'm not sure what we need this for (especially if we move the default methods to TransactionSetting) but I don't think  an interface should be called `....Impl`. If we need some sort of base implementation class it should be an abstract class, not an interface. It should also be called AbstractTransactionSetting to stay consistent with naming conventions elsewhere in the projbect.

Reply:
Was initially a class, but then I switched to enum in the ShaclSail and had to make it an interface. Might just remove it.